### PR TITLE
Translate "breaking update" messages

### PR DESCRIPTION
**Pull request by javabalm**
## Description

Translate the "breaking upgrade" messages into all Wazuh languages available.

- Checked languages
- [x] Portuguese
- [x] Chinese
- [x] German
- [x] Greek
- [x] English
- [x] Spanish
- [x] French
- [x] Hungarian
- [x] Italian
- [x] Japanese
- [x] Dutch
- [x] Polish
- [x] Russian
- [x] Bosnian
- [x] Turkish

### CODE REVIEW

File: etc/templates/fr/messages.txt [hunk 1]
```diff
@@ -113,6 +113,11 @@ updatefailed="Echec de la mise à jour."
 updaterules="Voulez-vous mettre à jour les règles ?"
 updatingrules="Mise à jour des règles."
 notupdatingrules="Règles non mises à jour."
+breakingupgrade1="ATTENTION:"
+breakingupgrade2="Votre version actuelle de $NAME:"
+breakingupgrade3="Vous avez une version antérieure à la v5.0.0, si vous mettez à niveau, vous perdrez toute votre configuration."
+breakingupgrade4="Voulez-vous continuer?"
```
Comment (julienhaziza) [inline etc/templates/fr/messages.txt:119]: breakingupgrade4="Voulez-vous continuer ?"

**Review by julienhaziza — COMMENTED** [review]
Translating "Breaking update" messages correct typos in french

File: etc/templates/es/messages.txt [hunk 2]
```diff
@@ -112,6 +112,11 @@ updatefailed="La actualización ha fallado."
 updaterules="¿Desea actualizar las reglas?"
 updatingrules="Actualizando las reglas."
 notupdatingrules="No se actualizan las reglas."
+breakingupgrade1="ADVERTENCIA:"
+breakingupgrade2="La versión actual de $NAME:"
+breakingupgrade3="Usted tiene una versión anterior a v5.0.0, si actualiza perderá toda su configuración."
+breakingupgrade4="Quieres continuar?"
```
Comment (TomasTurina) [inline etc/templates/es/messages.txt:118]:
```suggestion
breakingupgrade4="¿Quieres continuar?"
```

File: etc/templates/fr/messages.txt [hunk 3]
```diff
@@ -113,6 +113,11 @@ updatefailed="Echec de la mise à jour."
 updaterules="Voulez-vous mettre à jour les règles ?"
 updatingrules="Mise à jour des règles."
 notupdatingrules="Règles non mises à jour."
+breakingupgrade1="ATTENTION:"
+breakingupgrade2="Votre version actuelle de $NAME:"
+breakingupgrade3="Vous avez une version antérieure à la v5.0.0, si vous mettez à niveau, vous perdrez toute votre configuration."
+breakingupgrade4="Voulez-vous continuer ?"
```
Comment (TomasTurina) [inline etc/templates/fr/messages.txt:119]:
```suggestion
breakingupgrade4="Voulez-vous continuer?"
```

**Review by TomasTurina — APPROVED** [review]

**Pull request merged**